sdk/turing/router/config/log_config.py

```diff
@@ -105,6 +105,7 @@ class InvalidResultLoggerTypeAndConfigCombination(Exception):
     pass
 
 
+@dataclass
```

Comment:
Is there a reason why the dataclass decorator is added?

Reply:
I did it to align its implementation with that of its child class `LogConfig`, which is also a `dataclass`. I'm not sure if your comment is similar to Terence's one above on the same file, but essentially I thought it'd be nice for users to get the free `__repr__` and `__eq__` implementations when manipulating these `LogConfig` derivatives.

Reply:
Ah ok, no it's not a trick question because I haven't used that decorator before.

Reply:
Ahaha okay xD